### PR TITLE
garbro: Add version 1.5.44.2904

### LINKED ISSUE #9611: [Request] GARBro
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** GARBro

**Description:** A tool for browsing a wide range of visual novel archive files through a GUI interface

**Homepage:** [GitHub](https://github.com/morkt/GARbro)

**Download link(s):** [GitHub Releases](https://github.com/morkt/GARbro/releases)

**Some indication of popularity/repute:** [>1k Stars, >150 Forks, GitHub](https://github.com/morkt/GARbro)

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #9611

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/garbro.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "license": "MIT",
+    "url": "https://github.com/morkt/GARbro/releases/download/v1.5.44/GARbro-v1.5.44.2904.rar",
+    "hash": "C0E64B881F967E0A1473127B1D181C0808068A47E99D8B5F1FBE2D99A6B2ED53",
+    "bin": [
+        "GARbro.GUI.exe",
+        [
+            "GARbro.GUI.exe",
+            "garbro"
+        ]
+    ],
```
Comment (THEshezzee) [inline bucket/garbro.json:14]: `bin` is excess, as it GUI software.
```suggestion
```

File: bucket/garbro.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.5.44.2904",
+    "homepage": "https://github.com/morkt/GARbro",
+    "description": "A tool for browsing a wide range of Visual Novel resources through a GUI interface.",
+    "license": "MIT",
+    "url": "https://github.com/morkt/GARbro/releases/download/v1.5.44/GARbro-v1.5.44.2904.rar",
+    "hash": "C0E64B881F967E0A1473127B1D181C0808068A47E99D8B5F1FBE2D99A6B2ED53",
```
Comment (THEshezzee) [inline bucket/garbro.json:7]: Pretty.
```suggestion
    "hash": "c0e64b881f967e0a1473127b1d181c0808068a47e99d8b5f1fbe2d99a6b2ed53",

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**